dev/lint.sh

```diff
@@ -12,12 +12,12 @@ black --check .
 if [ $? -ne 0 ]; then
   echo '
 To apply black foramtting, do one of the following:
-- Run `pip install $(cat .github/workflows/scripts/lint-requirements.txt | grep "^black==") && black .`
+- Run `pip install $(cat requirements/lint-requirements.txt | grep "^black==") && black .`
```

Comment:
Can we remove lint.sh too?

Reply:
Unfortunately we can't since the mlflow lint.sh doesn't take a customize pylintrc or working directory that it should run the lint check on.
So we get many errors:
```| ========== pylint ==========
| 
| ************* Module show_package_release_dates
| .github/workflows/scripts/show_package_release_dates.py:1:0: C0114: Missing module docstring (missing-module-docstring)
| .github/workflows/scripts/show_package_release_dates.py:3:0: E0401: Unable to import 'requests' (import-error)
| .github/workflows/scripts/show_package_release_dates.py:8:0: C0116: Missing function or method docstring (missing-function-docstring)
| .github/workflows/scripts/show_package_release_dates.py:27:0: C0116: Missing function or method docstring (missing-function-docstring)
| .github/workflows/scripts/show_package_release_dates.py:44:0: C0116: Missing function or method docstring (missing-function-docstring)
| .github/workflows/scripts/show_package_release_dates.py:48:0: C0116: Missing function or method docstring (missing-function-docstring)
| .github/workflows/scripts/show_package_release_dates.py:51:11: W0703: Catching too general exception Exception (broad-except)
| .github/workflows/scripts/show_package_release_dates.py:56:0: C0116: Missing function or method docstring (missing-function-docstring)
| .github/workflows/scripts/show_package_release_dates.py:4:0: C0411: standard import "from concurrent.futures import ThreadPoolExecutor" should be placed before "import requests" (wrong-import-order)
| .github/workflows/scripts/show_package_release_dates.py:5:0: C0411: standard import "import traceback" should be placed before "import requests" (wrong-import-order)
| ************* Module databricks
| notebooks/databricks.py:8:0: C0301: Line too long (127/100) (line-too-long)
| notebooks/databricks.py:9:0: C0301: Line too long (168/100) (line-too-long)
| notebooks/databricks.py:1:0: C0114: Missing module docstring (missing-module-docstring)
| notebooks/databricks.py:18:0: E0401: Unable to import 'mlflow.pipelines' (import-error)
| ************* Module ingest
| steps/ingest.py:29:4: R1705: Unnecessary "else" after "return" (no-else-return)
| steps/ingest.py:30:8: C0415: Import outside toplevel (pandas) (import-outside-toplevel)
| ************* Module split
| steps/split.py:24:4: C0103: Argument name "df" doesn't conform to snake_case naming style (invalid-name)
| ************* Module train
| steps/train.py:15:4: C0415: Import outside toplevel (sklearn.linear_model.SGDRegressor) (import-outside-toplevel)
| ************* Module tests.ingest_test
| tests/ingest_test.py:1:0: C0114: Missing module docstring (missing-module-docstring)
| tests/ingest_test.py:1:0: E0401: Unable to import 'pytest' (import-error)
| tests/ingest_test.py:10:0: C0116: Missing function or method docstring (missing-function-docstring)
| tests/ingest_test.py:16:45: W0621: Redefining name 'sample_data' from outer scope (line 10) (redefined-outer-name)
| tests/ingest_test.py:16:0: C0116: Missing function or method docstring (missing-function-docstring)
| tests/ingest_test.py:2:0: C0411: standard import "import os" should be placed before "import pytest" (wrong-import-order)
| tests/ingest_test.py:3:0: C0411: standard import "import tempfile" should be placed before "import pytest" (wrong-import-order)
| ************* Module tests.split_test
| tests/split_test.py:1:0: C0114: Missing module docstring (missing-module-docstring)
| tests/split_test.py:1:0: E0401: Unable to import 'pytest' (import-error)
| tests/split_test.py:9:0: C0116: Missing function or method docstring (missing-function-docstring)
| tests/split_test.py:15:58: W0621: Redefining name 'sample_data' from outer scope (line 9) (redefined-outer-name)
| tests/split_test.py:15:0: C0116: Missing function or method docstring (missing-function-docstring)
| tests/split_test.py:27:50: W0621: Redefining name 'sample_data' from outer scope (line 9) (redefined-outer-name)
| tests/split_test.py:27:0: C0116: Missing function or method docstring (missing-function-docstring)
| tests/split_test.py:2:0: C0411: standard import "import os" should be placed before "import pytest" (wrong-import-order)
| ************* Module tests.train_test
| tests/train_test.py:1:0: C0114: Missing module docstring (missing-module-docstring)
| tests/train_test.py:5:0: C0116: Missing function or method docstring (missing-function-docstring)
| tests/train_test.py:11:0: C0116: Missing function or method docstring (missing-function-docstring)
| tests/train_test.py:2:0: C0411: third party import "from sklearn.utils.estimator_checks import check_estimator" should be placed before "from steps.train import estimator_fn" (wrong-import-order)
| ************* Module tests.transform_test
| tests/transform_test.py:1:0: C0114: Missing module docstring (missing-module-docstring)
| tests/transform_test.py:4:0: C0116: Missing function or method docstring (missing-function-docstring)
| ```